atlasdb-dbkvs-hikari/src/main/java/com/palantir/nexus/db/pool/config/ConnectionConfig.java

```diff
@@ -57,6 +58,7 @@ public MaskedValue getDbPassword() {
 
     public abstract String getDriverClass();
 
+    @Nullable
```

Comment:
is there a reason to make this nullable? should we just treat empty as not using test query?

Reply:
Yeah that is a better idea and improves the real-world testability. But will split out entirely.